tests/integration/components/feed-item-test.js

```diff
@@ -14,7 +14,7 @@ module('Integration | Component | feed item', function(hooks) {
     const article = {
       author: {
```

Comment:
I'm not going to boil the ocean now but we should refactor this test to use actual models.